src/resources/html-attributes.ts

```diff
@@ -0,0 +1,17 @@
+// this function will ensure that we propertly treat a potential string value for a boolean attribute
+// as the boolean representation
+export function normalizeBooleanAttribute(attributeName: string, value: boolean | string): boolean {
```

Comment:
Sweet! Yeah! I didn't know that a comment in that style at that location would also be picked up by various editors.